backend/app/assets/javascripts/spree/backend/views/order/shipment_tracking.js

```diff
@@ -0,0 +1,43 @@
+Spree.Views.Order.ShipmentTracking = Backbone.View.extend({
+  tagName: 'tr',
+  className: 'edit-tracking',
+
+  events: {
+    "click .js-edit":   "onEdit",
+    "click .js-save":   "onSave",
```

Comment:
It's very inconvenient one can't submit all our Backbone views with Enter key. We should change that.

If we would listen to `submit form` events instead and wrap the fields into a `form` (and use that magic submit button trick shown above) we can have both behaviours (enter key and click on the button) with the same event handler.

I will submit a PR with the changes to your PR.

Reply:
👍

Reply:
We can't listen to `submit form` because we can't wrap only one table row in a `<form>`.